cameronRomo/silver-nest
# Persist on apply

### What’s this PR do?
- Media queries for size "sm" were added to About page so that it is now viewable in mobile and various devices.
- Heart now persists after a student applies to live with a resident, exits the profile view, and comes back to the resident within the same semester.
- Text is aligned left in profile view for better readability
### Where should the reviewer start?
- About, line 16
- Profile, line 84
### How should this be manually tested?
- Run ` npm start` , open dev tools, view About page on multiple devices. Play with the responsive view.
-  Click "Apply" button in profile view, select "Agree". See heart appear and "Application Pending" message. Button should now be disabled. Exit Profile, scroll through other residents. Come back to applied resident's profile, and heart/application pending should still be there. 
### Any background context you want to provide?
For the future: Consider adding an "applied" property that takes a boolean to migration in database, edit seed file accordingly
### Screenshots (if appropriate):
![About Page "sm" Media Query](https://user-images.githubusercontent.com/68264128/104768851-8d412880-572b-11eb-9c1c-0cadb43e7ac4.png)
![Left-align Resident Info Text](https://user-images.githubusercontent.com/68264128/104769040-e315d080-572b-11eb-9380-dfe5ca9e14db.png)
### Questions:
n/a
